ide/dune-workspace

```diff
@@ -0,0 +1,6 @@
+(lang dune 1.2)
+
+; Add custom flags here. Default developer profile is `dev`
+(env
+ (dev     (flags :standard -rectypes -w -9-27-50+60))
+ (release (flags :standard -rectypes)))
```

Comment:
Isn't it inherited from `dune-workspace` if absent?

Reply:
Not if `-p coqide` is used [release, isolated build]. Note that there is a problem indeed with how `-rectypes` are handled, it should not be necessary that we add them here.

I believe this problem is in both `ocamlfind` and Dune. I propose we implemented this while I discuss the `-rectypes` issue upstream.

Reply:
@Zimmi48 so I had some discussion with Dune upstream and indeed they suggested a better layout that should avoid this redundancy problem. Will update once we converge I guess in a subsequent PR.